sources/lib/reasoners/fun_sat.ml

```diff
@@ -75,7 +75,7 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
         SE.fold
           (fun f mp ->
              let w = var_inc +. try ME.find f mp with Not_found -> 0. in
-             stable := !stable && Pervasives.(<=) w 1e100;
+             stable := !stable && Stdlib.(<=) w 1e100;
```

Comment:
Float comparison ? (By the way, do we care about the behaviour of this comparison on nan ? because iirc there are some tricky differences between compare and infix comparisons on floats)

Reply:
Stdlib.(infix op) by Stdlib.compare